### PR TITLE
improved partial compile documentation for LSM

### PR DESCRIPTION
**Pull request by sanderr**
# Description

closes inmanta/inmanta-lsm#1132

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: docs/model_developers/resource_sets.rst [hunk 1]
```diff
@@ -10,6 +10,11 @@ Partial compiles
     Care should be taken to :ref:`implement this safely<partial-compiles-guidelines>`, and the user should be aware of
     :ref:`its limitations<partial-compiles-limitations>`.
 
+.. note::
+
+    For partial compiles through LSM, see :ref:`its documentation<partial_compile_lsm>` on how to manage resource sets for an
+    LSM service in addition the documentation below.
```
Comment (arnaudsjs) [inline docs/model_developers/resource_sets.rst:16]:
```suggestion
    LSM service in addition to the documentation below.
```

Comment (arnaudsjs) [inline docs/model_developers/resource_sets.rst:16]: This link references a blank page on the OSS documentation.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: docs/model_developers/resource_sets.rst (hunk 1 above)
Comment (sanderr) [reply docs/model_developers/resource_sets.rst]: I made it so the note is only visible on the iso docs.

Comment (arnaudsjs) [reply docs/model_developers/resource_sets.rst]: I think you forgot to push that change

Comment (sanderr) [reply docs/model_developers/resource_sets.rst]: So I did. Pushed now.

**Review by arnaudsjs — APPROVED** [review]

**Review by DariushMoh — APPROVED** [review]

**Review by hlloreda — APPROVED** [review]

**Pull request closed without merging**